.github/workflows/build.yml

```diff
@@ -112,3 +112,8 @@ jobs:
           tags: |
             "${{ env.CONTAINER_REGISTRY }}/${{ steps.build_variables.outputs.REPO }}:${{ github.ref_name }}-latest-java11-unvalidated-ubuntu"
             "${{ env.CONTAINER_REGISTRY }}/${{ steps.build_variables.outputs.REPO }}:${{ steps.build_variables.outputs.VERSION }}-java11-unvalidated-ubuntu"
+
+      - name: Test local slim container image
```

Comment:
pr.yml also needs this.

Reply:
Thanks will add that :)

Reply:
Yeah :) That's that kork config with the default locations if I recall right.  Will dig & fix here later.

Reply:
https://github.com/spinnaker/kork/blob/2b857f57c85a8c821e35da68c1207fbf00cb335d/kork-config/src/main/java/com/netflix/spinnaker/kork/boot/DefaultPropertiesBuilder.java#L32C1-L33C1 this was what was thinking.  Checking on the config here.

Reply:
FYI fixed this with latest PR, but working through all the "clouddriver isn't available" as part of the integration tests.  Got some more work to do on the downstream deps.